api/server/server.go

```diff
@@ -52,6 +52,7 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 
 		if err := handlerFunc(ctx, w, r, vars); err != nil {
 			statusCode := httpstatus.FromError(err)
+			log.G(ctx).Debugf("Handler for %s %s returned error: %v", r.Method, r.URL.Path, err)
 			if statusCode >= 500 {
 				log.G(ctx).Errorf("Handler for %s %s returned error: %v", r.Method, r.URL.Path, err)
 			}
```

Comment:
I guess this one should be "either" error or debug, but not both?

Reply:
Also wondering if this would be the right moment to change these to use structured logs;

```go
log.G(ctx).WithFields(logrus.Fields{"error": err, "method": r.Method, "path": r.URL.Path}).Error("Handler for %s %s returned error")
```

WDYT @corhere ?

Reply:
I agree on both counts

Reply:
Converted those to structured logs and it would now be either error or debug